molo/forms/views.py

```diff
@@ -171,11 +179,12 @@ def get_context_data(self, *args, **kwargs):
                     question_stats = results.get(label, {})
                     question_stats[answer] = question_stats.get(answer, 0) + 1
                     results[label] = question_stats
-        if form.show_results_as_percentage:
-            for question, answers in results.items():
-                total = sum(answers.values())
-                for key in answers.keys():
-                    answers[key] = int((answers[key] * 100) / total)
+
+            if form.show_results_as_percentage:
```

Comment:
This has been moved inside the "if form.show_results:" block. Why?

Reply:
we don't need to execute that evaluation, if `form.show_results` is false